README.md

```diff
@@ -98,13 +100,14 @@ Go on [official Rails website](https://gorails.com/setup/windows/10) & follow th
 - [Active Record](https://guides.rubyonrails.org/active_record_basics.html) - Object Relational Mapping system
 
 ## Contributing
-Pull requests are welcome. Please make sure that your PR is [well-scoped](https://www.netlify.com/blog/2020/03/31/how-to-scope-down-prs/).
+Please read our [wiki on contributing](https://github.com/sylwiavargas/The-Code-Bending-Dictionary/wiki/3.-Contributing) and [code of conduct](https://github.com/sylwiavargas/The-Code-Bending-Dictionary/wiki/2.-Code-of-Conduct). 
```

Comment:
hey @sylwiavargas, I think this should be the link for the contributing wiki: https://github.com/sylwiavargas/The-Code-Bending-Dictionary/wiki/3.-Contributing-to-the-Code-Bending-Dictionary

Reply:
thank you, @bigfishh , great catch! I have just amended my last commit.